modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -219,6 +219,72 @@ class NDB_Form_User_Accounts extends NDB_Form
             unset($values['Password_md5']);
         }
 
+        // multi-site UPDATE
+        $uid = $user->getData('ID');
+
+        $us_curr_sites =array();
```

Comment:
From this line to line #285, all the code could be replace by: 
`DELETE FROM user_psc_rel WHERE UserID=:uid`
followed by `$centerIDs = implode(','$values[CenterIds])` then `INSERT INTO user_psc_rel SELECT $uid as UserID, CenterID from psc where FIND_IN_SET( CenterID, $centerIDs)`